pkg/controllers/cstorvolumeconfig/volume_operations.go

```diff
@@ -460,7 +461,7 @@ func (c *CVCController) createCVR(
 	// Set isRestoreVol annotation on CVR if CVC has
 	// "openebs.io/created-through: restore" annotation
 	if value := claim.GetAnnotations()["openebs.io/created-through"]; value == "restore" {
-		annotations["isRestoreVol"] = "true"
+		annotations[volumereplica.IsRestoreVol] = "true"
```

Comment:
:100:

Reply:
`openebs.io/created-through` ---> This can be put into a constant. 
Usually -- these kinds of labels and annotations keys are put into openebs/api repo.
But we can put this particular thing in as a constant in the current file and later move it to openebs/api.

Reply:
Similarly, the value `restore` needs to be placed as a value constant. This can help avoid errors ( e.g. due to spelling mistakes if somecode want to use it again ) plus it helps to understand the possible available allowed values.

Reply:
done